### ISSUE TITLE
Inconsistent Results on BDD100K Tracking Validation Set

### ISSUE THREAD

**Issue by taheranjary** [SysCV/qdtrack#82]
Hi there.

I ran the pre-trained BDD100K model on the tracking validation set and the resulting MOTA IDF1 scores are lower than what QDTrack claim: MOTA: 54.5, IDF1: 66.7 vs your  MOTA: 63.5, IDF1 71.5.

Kindly verify if this is the case for you or if there are any missing settings. 

I followed the instructions and ran this command:
sh ./tools/dist_test.sh ./configs/bdd100k/qdtrack-frcnn_r50_fpn_12e_bdd100k.py ./ckpts/mmdet/qdtrack_frcnn_r50_fpn_12e_bdd100k_13328aed.pth 2 --out exp.pkl --eval track
